crates/move-stackless-bytecode/src/dynamic_field_analysis.rs

```diff
@@ -458,6 +459,12 @@ impl FunctionTargetProcessor for DynamicFieldAnalysisProcessor {
             return data;
         }
 
+        let info = get_info(&FunctionTarget::new(&fun_env, &data));
+        if !info.verified || !info.inlined {
```

Comment:
this is incorrect, as it skips functions that are not both `inlined` and `verified`. it should be `!info.verified && !info.inlined`